docker-for-mac/osxfs-caching.md

```diff
@@ -1,237 +0,0 @@
-`FSEvents`) are consistently propagated in both directions.
-
-On Linux, these guarantees carry no overhead, since the underlying VFS is
-shared directly between host and container. However, on macOS (and
-other non-Linux platforms) there are significant overheads to
-guaranteeing perfect consistency, since messages describing file system
-actions must be passed synchronously between container and host. The
-current implementation is sufficiently efficient for most tasks, but
-with certain types of workloads the overhead of maintaining perfect
-consistency can result in significantly worse performance than a
-native (non-Docker) environment. For example,
-
- * running `go list ./...` in the bind-mounted `docker/docker` source tree
-   takes around 26 seconds
-
- * writing 100MB in 1k blocks into a bind-mounted directory takes
-   around 23 seconds
-
- * running `ember build` on a freshly created (empty) application
-   involves around 70000 sequential syscalls, each of which translates
-   into a request and response passed between container and host.
-
-Optimizations to reduce latency throughout the stack have brought
-significant improvements to these workloads, and a few further
-optimization opportunities remain. However, even when latency is
-minimized, the constraints of maintaining consistency mean that these
-workloads remain unacceptably slow for some use cases.
-
-## Tuning with consistent, cached, and delegated configurations
-
-**_Fortunately, in many cases where the performance degradation is most
-severe, perfect consistency between container and host is unnecessary._**
-In particular, in many cases there is no need for writes performed in a
-container to be immediately reflected on the host. For example, while
-interactive development requires that writes to a bind-mounted directory
-on the host immediately generate file system events within a container,
-there is no need for writes to build artifacts within the container to
-be immediately reflected on the host file system. Distinguishing between
-these two cases makes it possible to significantly improve performance.
-
-There are three broad scenarios to consider, based on which you can dial in the
-level of consistency you need. In each case, the container has an
-internally-consistent view of bind-mounted directories, but in two cases
-temporary discrepancies are allowed between container and host.
-
- * `consistent`: perfect consistency
-   (host and container have an identical view of the mount at all times)
-
- * `cached`: the host's view is authoritative
-   (permit delays before updates on the host appear in the container)
-
- * `delegated`: the container's view is authoritative
-   (permit delays before updates on the container appear in the host)
-
-## Examples
-
-Each of these configurations (`consistent`, `cached`, `delegated`) can be
-specified as a suffix to the
-[`-v`](../engine/reference/run.md#volume-shared-filesystems)
```

Comment:
Are the `consistent`, `cached`,  and `delegated` options no longer supported? Wondering if we document them elsewhere @djs55 ?

Reply:
(I know some of these have been a "hot topic", so if we remove the docs about them, we should have a look at either adding to the FAQ ("do I still need `consistent`, `cached`, `delegated` ?)

Reply:
So is caching not a thing in FUSE driver?